docs/user-guide/how-to-use-header-only-layoutbuilder.md

```diff
@@ -48,21 +48,7 @@ The following cpp-headers are needed to use Layout Builders to use the header-on
 3. LayoutBuilder.h
 4. utils.h
 
-It is recommended to download these headers from the [release artifacts on GitHub](https://github.com/scikit-hep/awkward/releases)
-
-Awkward Array can be installed from PyPI using pip:
-
-```shell
-pip install awkward
```

Comment:
also, I wonder why not `python -m awkward-header-only.config --cflags` if `awkward` is already installed?

Reply:
The original motivation was to decouple the header-only libraries from the awkward packages, as they don't need to be versioned together. In practice, they will be, because `awkward-cpp` has to be updated whenever the header-only libraries are modified. Additionally, now that we don't have any Python linking, it seems odd to require a Python interpreter for a C++ project (although I suspect a significant proportion of these would want to be consumed by a Python extension, the set is not 1:1).

>  a flag for it is needed [for C++ 17]

I didn't account for that in my thinking when @jpivarski and I were looking at this. I don't think it's the end of the world; many C++ packages rely on documentation for this, but I agree that it was a benefit of providing an endpoint before.

I think @jpivarski weighed in on this, so keen to hear his thoughts too?

Reply:
> The original motivation was to decouple the header-only libraries from the awkward packages, as they don't need to be versioned together. In practice, they will be, because `awkward-cpp` has to be updated whenever the header-only libraries are modified. Additionally, now that we don't have any Python linking, it seems odd to require a Python interpreter for a C++ project (although I suspect a significant proportion of these would want to be consumed by a Python extension, the set is not 1:1).

The `header-only package` does not have any libraries and it is not part of `awkward-cpp`, but the `awkward` one, I think.